diff --git a/server_manager/Cargo.toml b/server_manager/Cargo.toml
index bafdb3f..a418ce8 100644
--- b/server_manager/Cargo.toml
+++ b/server_manager/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "server_manager"
-version = "1.0.4"
+version = "1.0.5"
 edition = "2021"
 
 [dependencies]
@@ -23,6 +23,7 @@ tower-http = { version = "0.5", features = ["fs", "trace"] }
 bcrypt = "0.15"
 tower-sessions = "0.12"
 time = "0.3"
+rpassword = "7.3"
 
 [dev-dependencies]
 criterion = "0.5"
